docs/ios_device_check.md

```diff
@@ -1,81 +1,158 @@
 
-    1.  Download the DeviceCheck Private Key file (`.p8`) and save it in a
-        secure location. **This file is a private key, so treat it like a
-        password.**
+   1. Download the DeviceCheck Private Key file (with a `.p8` file extension)
+      and save it in a secure location. **This file is a private key, you should
+      treat it like a password.**
 
-
-## Configuring DeviceCheck in the server
+## Configuring the server to use DeviceCheck
 
 To enable and configure Apple's DeviceCheck on the server, you must specify the
-following information for _each_ iOS application your server supports:
+following information for each iOS application your server supports:
 
 - **Team ID** - developer ID, used as the `iss` in the JWT.
 
 - **Key ID** - key ID, used as the `kid` in the JWT.
 
 - **Private Key** - private key (`.p8`), used to sign the JWT.
 
+## Sharing the DeviceCheck key with server operators
+
+You need to share these values with your server operator. If your server is
+operated by a third party, the preferred way to share these secrets is using
+[Secret Manager](https://cloud.google.com/secret-manager/docs/). The secret
+resides in a Google Cloud project that you control, and you can revoke access
+to the secret at any time.
+
+### Prerequisites
+
+- A [Google Cloud account](https://console.cloud.google.com/freetrial) and a
+  [project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+  When you create a project, keep a note of the project ID, you will need it in
+  this tutorial.
+
+- The [gcloud](https://cloud.google.com/sdk/install) command-line tool.
+
+- Your Team ID, Key ID, and Private Key in .p8 format. These values come from
+  the Apple Developer Portal using the instructions above.
+
+- The server operator's service account email. Your server operator will share
+  this value. This will be of the form `name@project.iam.gserviceaccount.com`.
+
+- A communication channel between yourself and your server operator, such as a
+  shared chat room, email, or a tele/video conference bridge.
+
+### Sharing the private key
+
+To share the DeviceCheck Private Key with a server operator:
+
+1. If you have not already done so, authenticate the gcloud CLI:
+
+   ```console
+   gcloud auth login && gcloud auth application-default login
+   ```
+
+   This will open two browser windows and ask you to authenticate with your
+   Google account. Use the same account that owns the Google Cloud project.
+
+1. Enable the [Secret Manager service](https://cloud.google.com/secret-manager/docs)
+   on your Google Cloud project:
+
+   ```console
+   gcloud services enable secretmanager.googleapis.com \
+      --project "${PROJECT_ID}"
+   ```
+
+1. Create a secret and upload the private key into the secret:
+
+   ```console
+   gcloud secrets create "devicecheck-key" \
+      --project "${PROJECT_ID}" \
+      --replication-policy "automatic" \
+      --data-file PATH-TO-KEY-FILE
```

Comment:
Surround with brackets? Add a `# <--` comment after maybe?

Reply:
Having a bracket can cause confusion as to whether the value is wrapped in the bracket or not. The bracket-less, self descriptive placeholder variable name is the style we use across Google Developer documentation - https://developers.google.com/style/code-syntax
Also avoid comments in commands as many folks copy+paste commands directly into their shell.